src/vaadin-item-mixin.html

```diff
@@ -133,13 +152,15 @@
       }
     }
 
+    /** @private */
```

Comment:
Can be also marked as `@protected` (listener isn't bound and can be reused).
Not a blocker though.

Reply:
Done.